packages/amplify-category-auth/provider-utils/awscloudformation/triggers/PreTokenGeneration/PreTokenGeneration.map.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Why is this empty JSON needed? Is this populated with something?

Reply:
This will be populated with metadata when we add templates.